src/Essentials/src/Screenshot/Screenshot.shared.cs

```diff
@@ -17,17 +17,17 @@ public interface IPlatformScreenshot : IScreenshot
 	{
 #if ANDROID
 		Task<IScreenshotResult> CaptureAsync(Android.App.Activity activity);
-		Task<IScreenshotResult> CaptureAsync(Android.Views.View view);
+		Task<IScreenshotResult?> CaptureAsync(Android.Views.View view);
```

Comment:
One thing I noticed on all platforms if the view is a 0x0 or it has no visible size (before layout) then it does not render and returns null.